dissect/target/plugins/apps/remoteaccess/anydesk.py

```diff
@@ -15,26 +15,37 @@ class AnydeskPlugin(RemoteAccessPlugin):
 
     __namespace__ = "anydesk"
 
-    # Anydesk log when service (Windows)
-    GLOBS = [
-        "/sysvol/ProgramData/AnyDesk/*.trace",
+    # Anydesk logs when installed as a service
+    SERVICE_GLOBS = [  # Only if service is installed
```

Comment:
```suggestion
    SERVICE_GLOBS = [
```
Previous comment should already be sufficient.